ndcube/utils/cube.py

```diff
@@ -121,6 +121,9 @@ def get_crop_item_from_points(points, wcs, crop_by_values):
         Denotes whether cropping is done using high-level objects or "values",
         i.e. low-level objects.
 
+    keep_dims : `bool`
+        If true keep size 1 dimensions rather than dropping
```

Comment:
I find this a little clearer.
```suggestion
        If False, return item that will drop length-1 dimensions.  Otherwise item will keep length-1 dimensions.
```